PythonAPI/pycocotools/cocoeval.py

```diff
@@ -673,7 +676,9 @@ def analyze(self):
 
         # reset Dt and Gt, params
         self.params = prm
-        self.makeplot(self.params.recThrs, np.mean(ps, axis=2), 'overall-all', save_dir=self.params.outDir)
+        figures = self.makeplot(self.params.recThrs, np.mean(ps, axis=2), 
+                       'overall-all', save_dir=self.params.outDir)
+        self.analyze_figures.update(figures)
         if 'supercategory' in self.cocoGt.cats.values()[0]:
```

Comment:
This line is throwing an error for me (python3.6).

`TypeError: 'dict_values' object does not support indexing`

Converting to a list fixed the problem
`if 'supercategory' in list(self.cocoGt.cats.values())[0]`

Reply:
fixed